### DISCUSSION TITLE
请求增加快捷键

### DISCUSSION THREAD

**Discussion by Aik-R** [the1812/Bilibili-Evolved#4959] [功能建议 / Ideas]
1.细分弹幕开关：比如顶部、底部、滚动
2.画面相关：向左、向右旋转，镜像
3.视频倍速：加速减
大佬不考虑开放行为自定义吗

**Comment by Geeyun-JY3** [comment]
1. 之前有人提过相关的：#4883
2. 旋转重复：#2010
3. 有这快捷键，需要装个插件：[`快捷键扩展 - 视频倍速`](https://github.com/the1812/Bilibili-Evolved/blob/77ada2305f3a4fba1c63dbe19cf08914cc787f38/doc/features/features.md#%E5%BF%AB%E6%8D%B7%E9%94%AE%E6%89%A9%E5%B1%95---%E8%A7%86%E9%A2%91%E5%80%8D%E9%80%9F)

另外「开放行为自定义」是指什么？这些不够用吗：https://github.com/the1812/Bilibili-Evolved/blob/preview/registry/lib/components/utils/keymap/help.md

**Reply by Aik-R** [reply]
感谢回复，之前一直用的H5播放器快捷操作脚本因为B站不知改啥而失效了，因为用习惯了所以想着能不能增加下。
行为自定义说的是“自定义快捷键动作”

**Reply by Geeyun-JY3** [reply]
脚本支持自定义快捷键的动作都在[`快捷键扩展`](https://github.com/the1812/Bilibili-Evolved/blob/c6ace25ab831eda31eaa3b968cf830ea46778d9b/doc/features/features.md#快捷键扩展)里列出来了，还有几个动作需要装对应的插件，在线仓库中搜索「快捷键扩展」就能找到了。你要不直接说你还希望增加什么动作吧。

**Reply by Aik-R** [reply]
可以的话 1.细分弹幕开关 2.画面旋转和镜像

**Comment by the1812** [comment]
一直都开放自定义的, 你会写代码加插件就行

例子: [快捷键扩展 - 开关弹幕列表](https://github.com/the1812/Bilibili-Evolved/blob/master/registry/lib/plugins/utils/keymap-toggle-danmaku-list/index.ts)

**Reply by Aik-R** [reply]
ok,已解决
